src/test/java/org/candlepin/resource/ActivationKeyResourceTest.java

```diff
@@ -516,6 +517,30 @@ public void testAddActivationKeyContentOverrides() {
             .containsExactlyInAnyOrderElementsOf(overridesToAdd);
     }
 
+    @Test
+    public void testAddPoolToKeyFailsInSCAMode() {
+        Owner owner = new Owner()
+                .setId("test-id")
```

Comment:
Double indents here, again. May need to check your IDE's settings to avoid submitting these.

Reply:
done, i got new laptop and i forgot to export IDEA setting. And our profile what we are offering in our documentation is outdated.

i fixed my idents in IDE